docker-compose.yml

```diff
@@ -85,14 +85,9 @@ services:
     environment:
       - MONGODB_ADDRESS=mongodb
   mongodb:
-    image: bitnami/mongodb:5.0.10-debian-11-r3
+    image: mongo:4.4
```

Comment:
Should we downgrade this? We're running mongo 5 in staging and prod

Reply:
which version? I just pick this because this was the last version published on docker. 

I thought bitnami version wasn't following the version of the underlying app

Reply:
We're using `MongoDB server version: 5.0.10` atm